### PR TITLE
Fix #1424 : Password strength indication

### LINKED ISSUE #1424: Password strength indication
When a user fills SIGN UP form, 
- Observed behaviour
after the clicking on 'sign up', user is informed about the password being common or not secure.

- Expected Behaviour
there should be an indicator telling the user about his password strength as he types each character.

Moreover, we do not have any specifications for password except that it should consist of 8 characters,
for making it better we could impose certain conditions.

Would You Like to Take this Issue?
YES !

Thanks!

### PR DESCRIPTION
**Pull request by Obtin**
@aka-jain Please review the PR.

### CODE REVIEW

**Comment by trexdinosaur** [comment]
@Obtin I think for passwords it might not be the best idea to test if it has a specific set of characters. 

Even though it might be a bit of performance hit (a small one), libraries like https://github.com/dropbox/zxcvbn offer a bit more advanced algorithms to determine whether a password is truly secure by checking against commonly used passwords and repetitive characters.

**Comment by Obtin** [comment]
@trexdinosaur I agree it is indeed a good idea. Let's have a final say from mentors. .:-)
@aka-jain

File: frontend/src/js/controllers/authCtrl.js [hunk 1]
```diff
@@ -176,6 +176,63 @@
...
+
+      //Function To check password Strength
+        vm.checkStrngth = function(password) {
+            //TextBox left blank.  
+            var password_strength = document.getElementById("#password_strength");
+            if (pswd == 0) {
+                $("#password_strength").html("");
+                return;
+            }
+            //Regular Expressions.
```
Comment (aka-jain) [inline frontend/src/js/controllers/authCtrl.js]: Create a service for this expression check. @Obtin

**Comment by aka-jain** [comment]
@Obtin @trexdinosaur Agree with you guys but as we are not using password strength much in our app then it would be good too create our own service (a new task :D).
@Obtin try to create a service that will check the password strength. Also add the screenshot of your final result.

**Comment by aka-jain** [comment]
Any update @Obtin

**Comment by Obtin** [comment]
@aka-jain trying to fix it.

**Comment by Obtin** [comment]
I had some problem with my git as a result of which I had to setup my project again. I have created a new pull request in this regard updated with the requested changes. Thank You.

**Pull request closed without merging**